package.json

```
@@ -3,10 +3,18 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
+  
   "scripts": {
-    "dev": "pnpm run --parallel dev",
+    "build": "turbo run build",
+    "dev": "turbo run dev --parallel",
+    "lint": "turbo run lint",
```

you can use it /w out `run` command
```suggestion
    "build": "turbo build",
    "dev": "turbo dev --parallel",
    "lint": "turbo lint",
```